addons/project/controllers/portal.py

```diff
@@ -222,6 +226,40 @@ def portal_my_project_subtasks(self, project_id, task_id, page=1, date_begin=Non
             pager_vals['url_args'].update(filterby=filterby)
             pager = portal_pager(**pager_vals)
 
+            values.update({
+                'project': project_sudo,
+                'show_project': True,
+                'task': task_sudo,
+                'grouped_tasks': values['grouped_tasks'](pager['offset']),
+                'pager': pager,
+                'searchbar_filters': OrderedDict(sorted(searchbar_filters.items())),
```

Comment:
I think a simple dictionary is enough here, dictionaries are sorted by insertion order [since python 3.6](https://docs.python.org/3.6/whatsnew/3.6.html#new-dict-implementation)